### PR TITLE
move test requirements away from normal requirements

### PR DESCRIPTION
**Pull request by stepansnigirev**

### CODE REVIEW

**Comment by stepansnigirev** [comment]
@k9ert can you look into this one?
I think we should find a way to separate test dependencies away from the main pip installation. What would be the best way to do it?
The method I used here (moving to another requirements file) breaks CI...

File: test_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# requirements for testing
```
Comment (k9ert) [inline test_requirements.txt:1]: can't we call them reuqirements_tests.txt ?

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -11,7 +10,6 @@ git clone https://github.com/cryptoadvance/specter-desktop.git
 cd specter-desktop
 virtualenv --python=python3 .env
 source .env/bin/activate
-pip3 install -r requirements.txt
```
Comment (k9ert) [inline DEVELOPMENT.md:14]: Not sure why you remove it here. IMHO for Development, you need both?
You want to be able to run the tests locally.

**Comment by k9ert** [comment]
Ah, just realizing that this is already merged.

File: DEVELOPMENT.md (hunk 2 above)
Comment (stepansnigirev) [reply DEVELOPMENT.md:14]: It installs all the requirements with `pip3 install -e .` anyways

**Pull request merged**